meinberlin/apps/budgeting/templates/meinberlin_budgeting/includes/proposal_form.html

```diff
@@ -35,14 +35,20 @@ <h2>{% blocktrans %}Contact Information{% endblocktrans %}</h2>
 
     <div class="form-check">
         <label class="form-check__label">
-            {{ form.contact_storage_consent }}
+            {{ form.contact_storage_consent|add_error_attr:"aria-invalid:true" }}
```

Comment:
I needed so long to find this, but the django_widget_tweaks can do that! :partying_face: 
https://pypi.org/project/django-widget-tweaks/

If it's fine like this @phillimorland , I will also add the individual errors to the non_field_errors and then do the same in a4.